sites/shared/components/workbench/menu/core-settings/core-setting-sa-bool.js

```diff
@@ -21,7 +21,7 @@ const CoreSettingSaBool = props => {
       <SumButton onClick={toggle}>
         <SumDiv>
           <Deg />
-          <span>{ t('settings:sa.t') }</span>
+          <span>{ t('settings:sabool.t') }</span>
           <span className="ml-4 opacity-50">
```

Comment:
With the phrasing above, I think we can drop the yes/no.
I would add a translation string for `saSize` that reads `Seam allowance size` and use that for the other label